tailormap-components/projects/core/src/lib/feature-form/form/form.component.ts

```diff
@@ -105,16 +110,26 @@ export class FormComponent implements OnDestroy, OnChanges, OnInit {
     this.formDirty = result;
   }
 
+  public setFormEditting(editting){
+    this.store$.dispatch(FormActions.setFormEditting({editting: editting}));
+  }
+
   public newItem(evt) {
     const type = LayerUtils.sanitizeLayername(evt.srcElement.id);
     this.store$.select(selectFormConfigForFeatureType, type)
       .pipe(takeUntil(this.destroyed))
```

Comment:
This does not seem right... Creating a new item is an action by a user so this could be triggered multiple times. Each time you have a subscription which creates a new object. I think you need take(1) instead of takeUntil here